about_us.html

```diff
@@ -85,7 +85,7 @@ <h1 class="sectionHeading">BAMP</h1>
                 <div class="teamMember">
                     <div class="teamMember--photo"><img src="images/aboutus/team/ashley.jpg"></div>
                     <div class="teamMember--name">Ashley Cousin</div>
-                    <div class="teamMember--role">Office Manger/<br>Artist</div>
+                    <div class="teamMember--role">Office Manger /<wbr>Artist</div>
```

Comment:
this is good but we should have a space after the slash too, since the `<wbr>` doesn't add one.

currently it renders like this:

<img width="179" alt="Screen Shot 2020-11-04 at 2 47 44 PM" src="https://user-images.githubusercontent.com/733916/98176110-c3049e00-1eac-11eb-8439-448ddc3fa45c.png">

FYI, there are actually different conventions around this, but the Chicago manual of style prescribes spaces around slashes if the terms have spaces within them -- https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Punctuation/faq0096.html

(I think that looks best, personally, because it makes it less ambiguous as to whether the slash divides the whole phrase, or just the two words touching it. lol)